data/html/index.html

```diff
@@ -50,11 +54,15 @@
 <h2>Welcome to OpenIndiana Hipster!</h2>
 <ul>
   <li><a href="C/index.html" title="Default Language">Default Language</a></li>
-  <li><a href="de/index.html" title="Deutsch">Deutsch</a></li>
+  <li><a href="br/index.html" title="Português">Português</a></li>
+  <li><a href="de/index.html" title="Deutsche">Deutsche</a></li>
+  <li><a href="es/index.html" title="Español">Español</a></li>
   <li><a href="el/index.html" title="ελληνικά">ελληνικά</a></li>
   <li><a href="en/index.html" title="English">English</a></li>
   <li><a href="fr/index.html" title="Français">Français</a></li>
+  <li><a href="nl/index.html" title="Nederlands">Nederlands</a></li>
```

Comment:
No, I just Googled this stuff. I'm a mono-lingual American;-) Let me see what I can do to provide some consistency.